### ISSUE TITLE
use RMSE instead of MSE for keras reporting

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/auto_ml#246]
should be able to look at their loss functions, find the exponential one, and then take an exponent of 0.5
